internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -217,9 +217,11 @@ func (opts *Opts) PreRun(ctx context.Context, outWriter io.Writer) error {
 	opts.shouldRunLogin = false
 	opts.setTier()
 
+	defaultProfile := config.Default()
 	return opts.PreRunE(
 		opts.initStore(ctx),
-		opts.InitFlow(config.Default()),
+		opts.SyncWithOAuthAccessProfile(defaultProfile),
```

Comment:
2 way sync opts (like --gov) with default profile and only then run init operations (since these init operations might require configuration from either the cmd opts or the default profile)